creds.go

```diff
@@ -195,3 +196,25 @@ func (creds *Credentials) Cert() (cert *Certificate, err error) {
 	}
 	return
 }
+
+// LoadCredentials attempts to load a set of Taskcluster Credentials from the
+// environment using the standard Taskcluster environment variables.  This
+// function never fails, but might return a pointer to a nil
+// tcclient.Credentials struct.
+func LoadCredentials() *Credentials {
+	creds := &Credentials{}
+
+	if value, present := os.LookupEnv("TASKCLUSTER_CLIENT_ID"); present {
+		creds.ClientID = value
+	}
+
+	if value, present := os.LookupEnv("TASKCLUSTER_ACCESS_TOKEN"); present {
+		creds.AccessToken = value
+	}
+
+	if value, present := os.LookupEnv("TASKCLUSTER_CERTIFICATE"); present {
+		creds.Certificate = value
+	}
+
+	return creds
```

Comment:
This method could be simplified to:

```go
func LoadCredentials() *Credentials {
    return &Credentials{
        ClientID: os.Getenv("TASKCLUSTER_CLIENT_ID"),
        AccessToken: os.Getenv("TASKCLUSTER_ACCESS_TOKEN"),
        Certificate: os.Getenv("TASKCLUSTER_CERTIFICATE"),
    }
}
```

since if an env var isn't set, the resulting value would be the zero value, which is an empty string.